docs/src/main/asciidoc/metrics.adoc

```diff
@@ -0,0 +1,45 @@
+== Stackdriver Monitoring
+
+Google Cloud Platform provides a service called https://cloud.google.com/monitoring/[Stackdriver Monitoring], and https://micrometer.io/docs/registry/stackdriver[Micrometer] can be used with it to easily instrument Spring Boot applications for observability.
+
+Spring Boot already provides auto-configuration for Stackdriver. This module enables to auto-detect the `project-id` and `credentials`. Also, it can be customized.
+
+Maven coordinates, using <<getting-started.adoc#_bill_of_materials, Spring Cloud GCP BOM>>:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-gcp-starter-metrics</artifactId>
+</dependency>
+----
```

Comment:
Users would also need a dependency on actuator starter; otherwise `Clock` bean will not be autoconfigured, preventing our autoconfig from triggering.

```
    <dependency>
      <groupId>org.springframework.boot</groupId>
      <artifactId>spring-boot-starter-actuator</artifactId>
    </dependency>
```

Or should this be in the starter, since `spring-cloud-gcp-starter-metrics` is not useful without actuator?

Reply:
That's right, `actuator` is required. Even though the new starter will not make sense without the `actuator` dependency, I would prefer to let this clear in the docs so the developers are aware about what dependencies are declaring for this case. WDYT?

Reply:
Yes, I agree. Just add a mention of it to the refdoc.
Otherwise, looks great -- I tested the starter in a tiny sample app.